### PR TITLE
Update logiq namespaces, license@ addresses, EULA removals, and general Logiq -> Apica replacements

### PR DESCRIPTION
**Pull request by bdunavant**

### CODE REVIEW

File: getting-started/logiq-paas-deployment/README.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ To give you a head start with configuring your Apica Ascent deployment, we've pr
 You can pass the `values.yaml` file with the `helm install` command using the `-f` flag, as shown in the following example.
 
 ```
-helm install logiq --namespace logiq --set global.persistence.storageClass=<storage_class_name> logiq-repo/logiq -f values.small.yaml
+helm install apica-ascent --namespace apica-ascent --set global.persistence.storageClass=<storage_class_name> logiq-repo/logiq -f values.small.yaml
```
Comment (rparth) [inline getting-started/logiq-paas-deployment/README.md]: logiq-repo/logiq should be logiq-repo/apica-ascent

we should change the repo add step to reflect that as well

**Review by rparth — CHANGES_REQUESTED** [review]

**Comment by rparth** [comment]
I think we should also fork the actual repo under the apica github and deprecate the logiq repo/delete and documentation should be updated to reflect that.

File: getting-started/logiq-paas-deployment/README.md (hunk 1 above)
Comment (bdunavant) [reply getting-started/logiq-paas-deployment/README.md]: Done, along with some other updates.

**Review by pepisito — APPROVED** [review]

**Comment by pepisito** [comment]
Straight forward string replacements from logiq to apica-ascent.  Not verified/validated the helm execution portion but the comment blocks look good.

**Pull request merged**